### DISCUSSION TITLE
NASA ROTOR37

### DISCUSSION THREAD

**Discussion by billdafoam** [mdolab/dafoam#465] [General]
Hi, 
is it possible somebody to help or any hint ? 
Cheers 

+------------------------------------------------------------------------------+
| pyDAFoam Error: designSurfaceFamily is not a valid option name.            |
+------------------------------------------------------------------------------+

Traceback (most recent call last):
  File "runScript_v2.py", line 169, in <module>
    DASolver.addFamilyGroup(DASolver.getOption("designSurfaceFamily"), DASolver.getOption("designSurfaces"))
  File "/home/dafoamuser/dafoam/packages/miniconda3/lib/python3.8/site-packages/dafoam/pyDAFoam.py", line 3764, in getOption
    raise Error("%s is not a valid option name." % name)
dafoam.pyDAFoam.Error

**Comment by friedenhe** [comment]
You are using an old tutorial version. Please run this case using the latest tutorials and DAFoam versions.

**Reply by billdafoam** [reply]
Ok I will use 3.07 version, cheers.

**Reply by billdafoam** [reply]
Hi there, 
Although I updated it's version the same problem exists. 
Still do not understand why ?  Another hint ?

**Comment by friedenhe** [comment]
You are not using the latest runScript for this case. Use this one: https://github.com/DAFoam/tutorials/blob/main/Rotor37_Compressor/runScript_v2.py
